tests/security/apparmor_profile/apache2_changehat.pm

```diff
@@ -52,7 +52,7 @@ sub run {
     my ($self) = shift;
 
     # only skip on maintenace, not on dev
-    if (is_sle('>=15-SP6') && (check_var('BETA', '0') || !get_var('BETA'))) {
+    if (is_sle('>=15-SP7') && (check_var('BETA', '0') || !get_var('BETA'))) {
```

Comment:
This will make the test fail on 15-SP6 in maintenance.

Since the ticket says that the test can be skipped also in dev, I guess that `is_sle('>=15-SP6')` will be enough,

Reply:
the test has been broken since forever. We will also switch to selinux with sle16, so there's no point doing complex skip logic. Just skipping on sles 15.sp6 and greater is enough, regarding of the beta status